### PR TITLE
Add download link for etcd-manager-ctl, clarify restarting etcd

### PR DESCRIPTION
**Pull request by dzoeteman**
Added download info for etcd-manager-ctl and added docs for restarting etcd based on discussion in #7054.

### CODE REVIEW

File: docs/etcd/backup-restore.md [hunk 1]
```diff
@@ -58,8 +57,11 @@ etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/m
 etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/events restore-backup [events backup file]
 ```
 
-Note that this does not start the restore immediately; you need to restart etcd on all masters 
-(or roll your masters quickly). A new etcd cluster will be created and the backup will be 
+Note that this does not start the restore immediately; you need to restart etcd on all masters.
+You can do this with a `docker stop` or `kill` on the etcd-manager containers on the masters (the container names start with `k8s_etcd-manager_etcd-manager`).
```
Comment (angeloskaltsikis) [inline docs/etcd/backup-restore.md]: you can always add the `docker restart` for the pods

Comment (dzoeteman) [reply docs/etcd/backup-restore.md]: > you can always add the `docker restart` for the pods

In my mind, stopping them makes more sense, just letting the containers get back on their own. And it's also the way @justinsb told me to do it, if I remember correctly. A restart should technically work too though.

**Comment by joshbranham** [comment]
@dzoeteman mind fixing the conflict with `master` here?

**Comment by dzoeteman** [comment]
@joshbranham Done :)

**Pull request merged**